MDANSE_GUI/Src/MDANSE_GUI/Tabs/Views/PlotDataView.py

```diff
@@ -14,12 +14,13 @@
 #    along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 from typing import Union
-
+import time
```

Comment:
We imported this when we were performance testing, we can remove it now since it is not being used.